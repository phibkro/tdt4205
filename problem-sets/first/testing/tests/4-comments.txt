// This file contains tests
//Spacesaftercommentshouldnotmatter
go go go // This should be accepted
10: dx=10 dy=-5 go // This should also be accepted
dx=-1 dy=5 go /* This should fail! */
go go // dx=-4.2 should be accepted
//
go / We need two slashes
