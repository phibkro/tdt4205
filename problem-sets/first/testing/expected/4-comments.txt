Table filled!
line    1: accepted: // This file contains tests
line    2: accepted: //Spacesaftercommentshouldnotmatter
line    3: accepted: go go go // This should be accepted
line    4: accepted: 10: dx=10 dy=-5 go // This should also be accepted
line    5: error: dx=-1 dy=5 go /* This should fail! */
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^
line    6: accepted: go go // dx=-4.2 should be accepted
line    7: accepted: //
line    8: error: go / We need two slashes
~~~~~~~~~~~~~~~~~~~~~~^
